client/src/model/states/lead_data_state.ts

```diff
@@ -6,5 +6,6 @@ import { StatusState } from '@/model/states/status_state';
  */
 export interface LeadDataState {
   data?: LeadData;
-  status?: StatusState;
+  waterSystemStatus?: StatusState;
```

Comment:
Can you walk me through why we have both? Do we want to have separate errors or be in a different state when 1 fails vs both?

Reply:
For parcels we request both sets of data (water system + parcel level) and they don't return to the FE at the same time, so if we have just one status it will be made success from pending when the first api call returns (at least the way its currently set up, not sure how we would modify a single status to keep track of both calls with this pattern). We want to wait until both return to show any data though, since we will show one or the other depending on whether parcel data is present. Let me know if this doesn't make sense and we can chat over video.

Reply:
Can you leave a TODO to create an error when no rows are present and then determine how to have 2 unified status for lead data that is only set to success when both values came through?

Reply:
Done!